### ISSUE TITLE
FEATURE : Stacked bar chart graphs

### ISSUE THREAD

**Issue by dwinter3** [rashidkpc/kibana2#124]
To visually show something like:

Event Type = alert
Split by = host

over time. Each bar would be a stack of smaller bars, each smaller bar representing a hosts number of events of type alert that make the total for the given time slice. something I do in splunk all the time and would really like to see in kibana.

**Comment by rashidkpc** [comment]
mmm yes, I also want this feature. Shouldn't be hard to implement, flot supports stacked charts.

**Comment by dwinter3** [comment]
Yeah this http://people.iola.dk/olau/flot/examples/stacking.html is what I was looking for.  

David Winter | Director of Hosted Solutions
Zenoss, Inc. | Transforming IT Operations
dwinter@zenoss.com | +1.804.496.1731 | Skype: davewinter3

**Comment by zombiepig01** [comment]
Add my vote to this (of course, maybe I should learn ruby and help out =8-).   

Being able to do something like '@type:blar split by @fields.foo' and have different colored boxes for each value of @fields.foo would be extremely helpful.  Also, something like: 
- (@type:blar AND @fields.foo:123) && (@type:bing AND @fields.x:999)

and have the events appear as different colors.

Logstash rocks, kibana makes it rockier!
